op-guide/ansible-deployment.md

```diff
@@ -142,138 +138,241 @@ pd_servers
+        ```
+        ansible-playbook deploy.yml -k
+        ```
+    
+    5. Start the TiDB cluster.
 
+        ```
         ansible-playbook start.yml -k
+        ```
 
-**Note:** If an error containing "Permission denied" is displayed, you can use the `chmod` command to change the permissions of the specified file. For example:
+- Use the normal user with  to deploy TiDB.
 
-```
-chmod 777 tidb-ansible-master/scripts/grafana-config-copy.py
-```
+    > **Note**: Before the deployment, you should create the normal `tidb` user account and add the sudo privileges. The following example uses the `tidb` user account as the user who runs the service.
 
-## 6 Test the cluster
+    1. Edit the `inventory.ini` file.
 
-6.1 Use the MySQL client to connect to the TiDB cluster:
+        ```
+        ## Connection
+        # ssh via root:
+        # ansible_user = root
+        # ansible_become = true
+        # ansible_become_user = tidb
+        # ssh via normal user
+        ansible_user = tidb
+        ```
+    
+    2. Connect to the network and download TiDB binary to the Control Machine.
 
-          mysql -u root-h 172.16.10.1 -P 4000
+        ```
+        ansible-playbook local_prepare.yml
+        ```
+    
+    3. Initialize the system environment and modify the kernel parameters.
 
-**Note:** The TiDB service default port is 4000.
+        ```
+        ansible-playbook bootstrap.yml -k -K
+        ```
+        
+        If the remote connection using the normal user requires a password, add the `-k` (lower case) parameter. This applies to other playbooks as well:
+        
+        ```
+        ansible-playbook bootstrap.yml -k
+        ```
+        
+        The execution of this playbook requires root privileges. If a password is needed when the normal user gets root privileges from sudo, add the `-K` (upper case) parameter:
+        
+        ```
+        ansible-playbook bootstrap.yml -k -K
+        ```
+    
+    4. Deploy the TiDB cluster.
 
-6.2 Open a browser to access the monitoring platform:
+        ```
+        ansible-playbook deploy.yml -k
+        ```
+    
+    5. Start the TiDB cluster.
+
+        ```
+        ansible-playbook start.yml -k
+        ```
 
-        http://172.16.10.1:3000
+## Test the Cluster
 
-The default account and password: admin/admin.
+It is recommended to configure load balancing to provide uniform SQL interface.
 
-## 7. Perform Rolling Update
-The rolling update of the TiDB service does not impact the ongoing business. The environment with the minimum number of instances is as follows:
+1. Connect to the TiDB cluster using the MySQL client.
 
-+ 2 TiDB instances
-+ 3 TiKV instances
-+ 3 PD instances
+    ```
+    mysql -u root-h 172.16.10.1 -P 4000
+    ```
+    
+    > **Note**: The default port of TiDB service is 4000.
+
+2. Access the monitoring platform using a browser.
+
+    ```
+    http://172.16.10.1:3000
+    ```
+    
+    The default account and password: `admin/admin`.
```

Comment:
normally we use `:` to split username and password.
`/` is also acceptable.

The reason is:

in full URL scheme:

```
protocol://username:password@hostname:port/path/to/your/resource?queryparam=value&another_query_param=value2#fragment

http://admin:admin@127.0.0.1:3000/
```